py-polars/tests/unit/lazyframe/cuda/test_node_visitor.py

```diff
@@ -0,0 +1,83 @@
+from __future__ import annotations
+
+import typing
+from functools import lru_cache, partial
+from typing import TYPE_CHECKING, Any, Callable
+
+import polars as pl
+from polars._utils.wrap import wrap_df
+
+if TYPE_CHECKING:
+    import pandas as pd
+
+
+@typing.no_type_check
+def test_run_on_pandas() -> None:
```

Comment:
@wence FYI

Reply:
Sadly, my github ID has a hyphen in it :( (`@wence-`), thanks for the ping though!